### ISSUE TITLE
[ASK] Anyone else experiencing overall connection difficulties while using Ricochet?

### ISSUE THREAD

**Issue by taoeffect** [ricochet-im/ricochet#357]
There is [reason to believe](https://www.reddit.com/r/Bitcoin/comments/352a82/big_blocks_and_tor_gavin_andresen/cr0dqaa) that long-term use of Ricochet can lead to overall connection troubles as some ISPs may actively kill long-lived Tor connections.

I've been noticing issues with my Internet connection, but I am not certain that it is as a result of leaving Ricochet open for a while.

Am just curious if anyone else has noticed this? Please comment here if you have.

(Note, this ticket is related to #350 and #351.)

**Comment by delaRosa12345** [comment]
Yeah I've been having connection problems too, even getting disconnected from time to time. Right now my network's fine, but later on I might get the same problems. I'm not saying this happens every time I have Ricochet running, but I often experience the problem when it is running. I don't know if Ricochet 'causing the problem or something else.

**Comment by cybershambles** [comment]
My connection works 20% of the time... I've been trying to figure out the environmental issues
but it seems to be on multiple networks and multiple devices on multiple OSes.

I'd love to have a debug build or at worst, a bot that let's me confirm the messenger is working.
